pkg/operation/botanist/component/seedsystem/seedsystem.go

```diff
@@ -162,7 +162,6 @@ var gardenletManagedPriorityClasses = []struct {
 	value       int32
 	description string
 }{
-	{v1beta1constants.PriorityClassNameSeedSystemCritical, 999998950, "PriorityClass for Seed system components"},
```

Comment:
Now that this is no longer part of the `ManagedResource`, this `PriorityClass` will be deleted after the `gardenlet` comes up. Use `ignore` annotation?